src/state.h

```diff
@@ -62,7 +63,7 @@ struct Pool {
   void DelayEdge(Edge* edge);
 
   /// Pool will add zero or more edges to the ready_queue
-  void RetrieveReadyEdges(set<Edge*>* ready_queue);
+  void RetrieveReadyEdges(EdgeSet* ready_queue);
```

Comment:
Should this really be a pointer-to-container?

Wouldn't a reference be a more appropriate choice?

Reply:
https://google.github.io/styleguide/cppguide.html#Reference_Arguments

Reply:
Alright.